springql/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// This file is part of https://github.com/SpringQL/SpringQL which is licensed under MIT OR Apache-2.0. See file LICENSE-MIT or LICENSE-APACHE for full license details.
+
+#![doc = include_str!("lib.md")]
+#![deny(missing_debug_implementations, missing_docs)]
+#![deny(rustdoc::broken_intra_doc_links)]
+
+pub use springql_core::api::*;
+
+/// error and result types for SpringQL
+pub mod error {
```

Comment:
We see two `error` modules in doc.

- left: `cargo d` result
- right: https://docs.rs/springql-core/latest/springql_core/api/index.html

<img width="1689" alt="image" src="https://user-images.githubusercontent.com/498788/173971948-daa3cd84-c578-4e80-8ee4-89a22744cea5.png">

Reply:
seems mod `error` from 

- `springql::error` and (`pub mod error` in springql`)
- `springql-core::api::error` (re-exported `pub use springql_core::api::*`)